src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1023,14 +1023,29 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                         $newline = [Environment]::Newline
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://img.shields.io/static/v1.svg?label=chat&message=on%20gitter&color=informational&logo=gitter should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 403 (Forbidden).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```